geometrical_calculator_kuba.py

```diff
@@ -0,0 +1,393 @@
+            print("To calculate THE CIRCUMFERENCE of rectangular prism ((4 * a) + (4 * b) + (4 * c)) - press '2'")
+            choose = input("Choose operation...")
+            if choose == '1':
+                a = float(input("Enter the length of the first side of the rectangular prism :"))
+                b = float(input("Enter the length of the second side of the rectangular prism :"))
+                c = float(input("Enter the length of the third side of the rectangular prism :"))
+                rectangular_prism_obj = RectangularPrism(a , b , c)
+                area = rectangular_prism_obj.calcuclate_area()
+                geometrical_calculator_history.append((f"Rectangular prism area with sides {a}, {b}, {c}", area))
+                print("The AREA of the rectangular prism is:", area)
+                break
+            elif choose == '2':
+                a = float(input("Enter the length of the first side of the rectangular prism :"))
+                b = float(input("Enter the length of the second side of the rectangular prism :"))
+                c = float(input("Enter the length of the third side of the rectangular prism :"))
+                rectangular_prism_obj = RectangularPrism(a , b , c)
+                circumference = rectangular_prism_obj.calculate_circumference()
+                geometrical_calculator_history.append((f"Rectangular prism circumference with sides {a}, {b}, {c}", circumference))
+                print("The CIRCUMFERENCE of the rectangular prism is:", circumference)
+        except ValueError:
+            print("Invalid choice. Please enter '1' for area or '2' for circumference.")
+        else:
+            break
+            
+            
+
+def choose_shape():
+    while True:
+        print("Please choose a shape for operation:")
+        print("1 - Circle")
+        print("2 - Triangle")
+        print("3 - Square")
+        print("4 - Rectangle")
+        print("5 - Rhombus")
+        print("6 - Regular pentagon")
+        print("7 - Regular hexagon")
+        print("8 - Rectangular prism")
+        shape = input("Enter the number corresponding to the shape: ")
+        if shape in ['1', '2', '3', '4', '5', '6', '7', '8']:
+            return shape
+        else:
+            print("Invalid choice. Please enter a number between 1 and 8.")
+
+def show_history():
+    print("Operation  geometrical history:")
+    for x in geometrical_calculator_history:
+        print(x)
+        
+def save_to_csv(calculations ,  file_name = 'Geometric calculat history.csv'):    
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    full_path = os.path.join(current_directory , file_name)
+    with open(full_path , 'w') as csvfile:
+        csv_writer = csv.writer(csvfile)
+        for expression , result in calculations:
+            csv_writer.writerow([f"{expression}={result}"])
+
+        
+def main():
```

Comment:
Nie używajmy tu main. Później importując libke, będzie słabo wiadome co dokładnie robi to, jeżeli plik miałby nazwę np. Worker.py 

Nazwy jak najbardziej opisowe, mają same za siebie mówić co robią jak się je wywoła, więc tutaj by pasowało coś w stylu `geometrical_calculator()`